policyengine_us/parameters/gov/states/oh/tax/income/rates/agi.yaml

```diff
@@ -0,0 +1,43 @@
+description: OH individual tax on nonbusiness income
+brackets:
+  - threshold:
+      0000-01-01: 0
+    base_amount: 
+      0
+    rate:
+      2021-01-01: 0
+  - threshold: 
+      0000-01-01: 25_001
+    base_amount:
+      2021-01-01: 346.16
+    rate:
+      2021-01-01: 0.02765
```

Comment:
Is it 2021? I can't tell from the legal code, which AFAICT doesn't specify a year and only says it was last updated in 2022. Tax form should clarify.

Reply:
Good question. I based the year on the tax form screenshot in the issue itself #1500 but I'm going to find an actual link to the form to double check, and then will include that as a second reference per your comment above.

Reply:
As best as I can find here, 2021 is the latest IT form posted - https://tax.ohio.gov/individual/get-a-form

Reply:
Confirmed that the most recent published tax form is 2021, though it does note it was updated 1/22.